README.md

```diff
@@ -66,32 +66,23 @@ No need to download libusb separately or set `LIBUSB_ROOT`.
 pacman -S $MINGW_PACKAGE_PREFIX-{toolchain,cmake,libusb}
```

Comment:
Do you need to add a ninja installation to this pacman command too, or is that installed by default (or by one of these packages already)?

Reply:
Nope, ninja is one of the [dependencies of CMake in MSYS2](https://packages.msys2.org/package/mingw-w64-x86_64-cmake) so it will always be installed if CMake is installed.